core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/src/org/polarsys/capella/core/platform/sirius/ui/commands/PasteCommandHelper.java

```diff
@@ -52,9 +51,8 @@ public class PasteCommandHelper {
   private PasteCommandHelper() {
     // Avoid to instantiate.
   }
-  
-  public static IStatus createPasteCommands(Collection<?> pasteElements, CompoundCommand commands, EObject owner, EStructuralFeature feature,
-      EditingDomain domain, int index, boolean useIndex) {
+
+  public static IStatus createPasteCommands(Collection<?> pasteElements, CompoundCommand commands, EObject owner, EStructuralFeature feature, EditingDomain domain, int index, boolean useIndex) {
```

Comment:
can you use capella formating, https://github.com/eclipse/capella/wiki/How-to-contribute#code

Reply:
ok but in the first place, the value should be correct in `.\releng\plugins\org.polarsys.capella.settings\JavaFormatter.xml` that I used(see `<setting id="org.eclipse.jdt.core.formatter.lineSplit" value="200"/>`  200 instead of 120). It would then be unnecessary to override it in the preference file.

Reply:
There are still differences on the non changed code but it seems coherent with the config.